frontend/components/customizer/customizer.js

```diff
@@ -129,24 +124,37 @@ export default class Customizer extends React.Component {
     this.setState({ queryString })
   }
 
+  constructor(props) {
+    super(props)
+    const { link } = { ...this.props.exampleQueryParams }
+    this.state = {
+      link: link || '',
```

Comment:
I think this is now stored in `state.queryString.link`, and if it's edited by the user, that's the value that will be updated. Is that right? Should that replace `this.state.link`?

Reply:
There are two `links` actually and we mixed them by moving link to `queryParams` :-)
- the first `link` was is used in markdown markup to link badge image to some resource https://github.com/badges/shields/blob/07a8d2ecbe8b178a4612e698deb275cf19ed096e/frontend/components/customizer/customizer.js#L77-L82 in https://user-images.githubusercontent.com/1526000/56317336-c1557700-615c-11e9-98b3-c25b5eb0fec4.png (the last working version https://github.com/badges/shields/commit/7955f460f37f7cfaa1e9588cb84aa050f2f14275) we had a separate input for link used as a markup link
- the second `link` is a query param used as a link in svg badge

Which one `link` should be the suggested link?

Reply:
Ahhh I see! That makes a lot of sense! Sorry for forgetting these were two different things.

Frankly I don't think the embedded SVG links are helpful except in very specific situations. So the suggested link should be used in the markdown, not in the query params.